src/cata_utility.cpp

```diff
@@ -193,7 +193,7 @@ const char *volume_units_abbr()
     if( vol_units == "c" ) {
         return _( "c" );
     } else if( vol_units == "l" ) {
-        return _( "L" );
+        return pgettext( "Abbreviation for liter", "L" );
```

Comment:
Generic contexts (say, `"volume_unit"` and `"mass_unit"`) would be lots better than sporadic ones.

Reply:
Sorry, I don't quite understand what do you mean.

Reply:
I meant specifying the same context for all the units:
```c++
    if( vol_units == "c" ) {
        return pgettext( "volume_unit", "c" );
    } else if( vol_units == "l" ) {
        return pgettext( "volume_unit", "L" );
    } else {
        return pgettext( "volume_unit", "qt" );
    }
```

Reply:
Use "volume_unit" for volumes and "mass_unit" for masses.